medical-portal/src/API/RSBC.DMF.MedicalPortal.API/ViewModels/ChefsSubmission.cs

```diff
@@ -11,8 +11,9 @@ public static class SubmissionStatus
 
 public class ChefsSubmission
 {
-    [JsonProperty("status")]
-    public string Status { get; set; } = SubmissionStatus.Draft;
+    [JsonProperty("status")] public string Status { get; set; } = SubmissionStatus.Draft;
```

Comment:
I thought the deserializing would work without the casing matching? There is a way to change the serializer to change the casing, we have some examples in other projects. No changes needed, just a heads up. For next time, I would try to match the existing code.